cdap-standalone/src/main/java/co/cask/cdap/StandaloneMain.java

```diff
@@ -386,6 +402,7 @@ public static void main(String[] args) {
         System.err.println("Failed to start Standalone CDAP");
         e.printStackTrace(System.err);
       }
+      main.stopExternalProcesses();
       Runtime.getRuntime().halt(-2);
```

Comment:
i know you didn't add this, can we add comments on the code used, -2 what does it mean ?